src/Tooltip.re

```diff
@@ -20,7 +18,20 @@ external make:
     ~className: string=?,
     ~content: 'content=?,
     ~position: Js.t({..})=?,
-    ~cursor: TooltipCursor.t=?,
+    ~cursor:
+      [@mel.unwrap] [
+        | `Bool(bool)
+        | `Obj(
+            Js.t({
```

Comment:
For closed object type, the definition is just `{. "fill": option(string) ...}` without the preceding `Js.t`. I'll fix it in a bit.